torch/_torch_docs.py

```diff
@@ -9350,6 +9350,8 @@ def merge_dicts(*dicts):
     tensor([[ 0.4875,  0.3938],
             [ 0.9158, -0.6929],
             [-0.5872,  0.6932]])
+
+See also :func:`torch.transpose`.
```

Comment:
We actually have "seealso" sections for this that appear before args. See https://pytorch.org/docs/master/generated/torch.linalg.svd.html?highlight=linalg%20svd#torch.linalg.svd for an example how do this. I would move both of these sentences into those sections for clarity + consistency.